### PR TITLE
Update osm port

### LINKED ISSUE #1548: [Meshery] Docker Compose: OSM and Tanzu SM Adapters are swapping network port assignments
**Current Behavior**
The `docker-compose.yml` file used to run Meshery and its adapters locally needs to be updated to reflect recent [network port reservations](https://meshery.layer5.io/docs/architecture#network-ports).

**Desired Behavior**
- [ ] OSM should be using 10009.
- [x] Tanzu SM should be using 10010.

**Implementation**
1. Update the OSM entry in Meshery's [docker-compose.yml](https://github.com/layer5io/meshery/blob/master/docker-compose.yaml#L62)

### PR DESCRIPTION
**Pull request by ratatatCat**
Signed-off-by: ratatatcat <iretoncat@gmail.com>

**Description**

This PR fixes #1548

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
@nikita15p, your review on this PR is most welcome.

**Review by kushthedude — APPROVED** [review]
Great work @ratatatCat !

File: docker-compose.yaml [hunk 1]
```diff
@@ -35,7 +35,7 @@ services:
     labels:
       - "com.centurylinklabs.watchtower.enable=true"
     ports:
-      - "10003:10003"
```
Comment (nikita15p) [inline docker-compose.yaml:38]: Can u undo this change

File: docker-compose.yaml [hunk 2]
```diff
@@ -47,7 +47,7 @@ services:
     labels:
       - "com.centurylinklabs.watchtower.enable=true"
     ports:
-      - "10007:10007"
```
Comment (nikita15p) [inline docker-compose.yaml:50]: Same as https://github.com/layer5io/meshery/pull/1565/files#r485856855

**Review by nikita15p — COMMENTED** [review]
few minor comments. otherwise lgtm.

File: docker-compose.yaml (hunk 1 above)
Comment (leecalcote) [reply docker-compose.yaml:38]: Good highlight, @nikita15p. Ideally, this change is reverted, but if doing so is a challenge, @ratatatCat, just signal. It's an inconsequential change and so, ultimately, we can move forward even with it here.

**Review by leecalcote — APPROVED** [review]
Thank you for jumping in here, @ratatatCat. 👍

**Comment by leecalcote** [comment]
@nikita15p, great review. Thank you for this.

**Review by mgfeller — APPROVED** [review]
Thanks for your contribution, @ratatatCat !

**Review by nikita15p — DISMISSED** [review]
@ratatatCat Thanks for the contribution!

File: docker-compose.yaml (hunk 1 above)
Comment (ratatatCat) [reply docker-compose.yaml]: Fixed!

File: docker-compose.yaml (hunk 2 above)
Comment (ratatatCat) [reply docker-compose.yaml]: Fixed!

**Review by kushthedude — APPROVED** [review]

**Comment by kushthedude** [comment]
Great work @ratatatCat
